FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -218,9 +221,12 @@ extension EncryptedStorage: PassPhraseStorageType {
         try updateKeys(with: passPhrase.primaryFingerprintOfAssociatedKey, passphrase: nil)
     }
 
-    func getPassPhrases() throws -> [PassPhrase] {
-        return try storage.objects(KeypairRealmObject.self)
-            .compactMap(PassPhrase.init)
+    func getPassPhrases(for email: String?) throws -> [PassPhrase] {
```

Comment:
Let's make `email` required. I don't see any reason to be pulling across accounts. Should always be for a particular account.